.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
you can add that to your local `.git/info/exclude` file, so we don't have OS specific things here.